src/app/navbar/navbar.component.html

```diff
@@ -26,6 +26,9 @@
         <a mat-button class="icon" (click)="resetPageNumber()" routerLink="/workflows">
             <img class="site-icons-small hidden-sm" src="../assets/images/dockstore/dockstore-workflows-purple.png"> Workflows
         </a>
+        <a mat-button class="icon" (click)="resetPageNumber()" routerLink="/organizations">
+          <mat-icon>people</mat-icon> Organizations
```

Comment:
Are you going with the US spelling for organi*z*ation? I prefer it, being from the US, but I noticed Andrew was using the British/Canadian spelling for the API, with an s. We should probably eventually be consistent.

:)

Reply:
:canada:

Reply:
Looks like the UK uses both. I guess we'll go with the zed
https://writingexplained.org/organisation-vs-organization-difference

Reply:
Do we want to change the API to use the zed as well? Not sure if that was implied in your comment or not, or if it's necessary.

Reply:
Do we want to change the API to use the zed as well? Not sure if that was implied in your comment or not, or if it's necessary.

Reply:
Organisation and Organization is both accepted in most places of the world.  Only Organization is accepted in the US.  I also prefer Organization.  If there is no upside to using Organisation, I'd rather use Organization.